spec/fixtures/users.yml

```diff
@@ -84,3 +84,14 @@ conf_admin:
   public_key: <%= "!!binary \"#{FixturesHelper.read_public_key('conf_admin')}\"" %>
   private_key: <%= "!!binary \"#{FixturesHelper.read_private_key('conf_admin')}\"" %>
 
+api_tux:
```

Comment:
why do we need an API fixture user here?
can't we just fabricate one where used in tests?

Reply:
We noticed, that we don't need it here in domain class, because the tests for api_user and non existent user are made in controller